lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -812,6 +812,9 @@ def post(self):
             application/json:
               schema:
                 type: object
+                required:
+                  - replicas
+                  - reason
```

Comment:
This `required` is technically not correct, since default values are present. I could call the endpoint without the parameters and everything is fine.

Either we keep this and remove the default values/add an exception if none are given, or they should not be declared as required...